test/dlc_tests/container_tests/bin/pytorch_tests/setupPyTorchBackendTest

```diff
@@ -10,10 +10,8 @@ sed -i '/from apex.fp16_utils import network_to_half, prep_param_lists, model_gr
 sed -i '/try:[ ]*$/{N;s/try:\s*from apex.parallel import DistributedDataParallel as DDP/from torch.nn.parallel import DistributedDataParallel as DDP/}' dlbs/python/pytorch_benchmarks/benchmarks.py
 sed -i 's/traceback.print_exc(file=sys.stdout)/traceback.print_exc(file=sys.stdout)\n        raise/g' dlbs/python/pytorch_benchmarks/benchmarks.py
 
-python -c "import sys;
-assert sys.version_info < (3, 7);"
-RETURN_VAL=`echo $?`
-if [ ${RETURN_VAL} -eq 1 ]; then
+RETURN_VAL=$(python -c "import sys; print(int(sys.version_info < (3, 7)));")
+if [ ${RETURN_VAL} -eq 0 ]; then
```

Comment:
The condition changed from equivalence to 1 to equivalence to 0. Can you confirm that the if statement code block should execute only when the system version is >= 3.7?

Reply:
yes this has been tested